HIPs/baf.md

```diff
@@ -0,0 +1,248 @@
+* [Blockchain Automation Framework Lab Proposal](https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/102)
+* [Blockchain Automation Framework GitHub Repo](https://github.com/hyperledger-labs/blockchain-automation-framework)
+* [Blockchain Automation Framework Read the Docs](https://blockchain-automation-framework.readthedocs.io/en/latest/index.html)
+* [Blockchain Automation Framework Chat Channel (#blockchain-automation-framework](https://chat.hyperledger.org/channel/blockchain-automation-framework))
+* [Blockchain Automation Framework Wiki Page](https://wiki.hyperledger.org/display/labs/Blockchain+Automation+Framework+lab)
+* [Blockchain Automation Framework Good First Issues](https://github.com/hyperledger-labs/blockchain-automation-framework/issues?labels=good-first-issue&is:open&q=is:issue)
+* [Blockchain Automation Framework Roadmap](https://blockchain-automation-framework.readthedocs.io/en/latest/roadmap.html)
+* [Blockchain Automation Framework Releases](https://github.com/hyperledger-labs/blockchain-automation-framework/releases)
+* [LF Insights Commit Report through July 18, 2021](https://tinyurl.com/yfckjkeu)
+
+
+# Closure
+<mark>_**Closure** how do we know that the project succeeded. This has to be measurable if possible. Make references to successor projects if any._</mark>
+
+Project success can be measured by the use of BAF to accelerate the deployment of production-ready DLT networks and nodes and  drive the adoption of DLT platforms.
+
+# FAQs
+Additional [FAQs](https://blockchain-automation-framework.readthedocs.io/en/latest/faq.html) can be found in the Blockchain Automation Framework documentation.
+
+## What is the Blockchain Automation Framework?
+The Blockchain Automation Framework is an automation framework for delivering consistent production ready DLT networks on cloud based infrastructures.
+
+## How is Blockchain Automation Framework different from Blockchain as a Service (BaaS)?
+* The Blockchain Automation Framework deployment scripts can be reused across cloud providers like AWS, Azure, GCP, DigitalOcean and OpenShift
+* Can deploy networks and smart contracts across different DLT/Blockchain platforms
+* Supports heterogeneous deployments in a multi-cloud, multi-owner model where each node is completely owned and managed by separate organizations
+* Bring Your Own Infrastructure (BYOI) - You provide GIT, Kubernetes cluster(s), and Hashicorp Vault services provisioned to meet your specific requirements and enterprise standards
+* No network size limit
+* Specifies only the number of organizations and the number of nodes per organization in a network.yaml file uniquely designed in the Blockchain Automation Framework for a new DLT/Blockchain network set-up and its future maintenance
+* Provides a sample supply chain application which runs on multiple DLT/Blockchain platforms that can be used as a reference pattern for how to safely abstract application logic from the underlying DLT/Blockchain platform
```

Comment:
Can we add the list of aspects inherent to BaaS but are not covered in BAF?
- consortium governance
- ongoing software upgrades while the network is running, including complex orchestration procedures like hard forks (such as for Ethereum based networks)
- monitoring metrics (both low-level resources metrics as well as DLT-level consensus liveness checks)
- backup and recovery
- 24/7 support

Reply:
I think it is more interesting to consider how BAF differs from BaaS https://github.com/hyperledger/hyperledger-hip/blob/0e70c4b6ae99cd8d788a1110162552a8eff58471/HIPs/baf.md#how-is-blockchain-automation-framework-different-from-blockchain-as-a-service-baas

Reply:
My bad. This is that section. Should not review on a phone. We could add that here.

Reply:
- Monitoring metrics (using prometheus or splunk)  is in the roadmap.
- Software upgrades like upgrade from Fabric 1.4.3 to 2.2.2, will be added in the roadmap as it has been planned as a feature
- Backup and recovery is provided by the underlying Infrastructure as we are saying Bring-your-own-infra

Reply:
> Software upgrades like upgrade from Fabric 1.4.3 to 2.2.2, will be added in the roadmap as it has been planned as a feature

this one was not about support for a specific ledger version, but to support seamless upgrade from one version to another at the user's discretion. Using an example: say I have used BAF to provision an Ethereum network with Besu version X, and would like to upgrade to version Y which supports the London hard fork, the following must happen for a smooth upgrade:
- all node owners agree on a future block number for the hardfork to become effective
- all nodes configurations get re-initialized with an updated genesis block that specifies the agreed block number for the London hardfork EIP
- all nodes must be upgraded before the chain height gets to the agreed block number
- the upgrade process be conducted in a rolling fashion to allow the network to continue to serve client applications during the upgrade

Reply:
@sownak thanks for clarifying. Glad to see that monitoring is in the roadmap 👍 . I agree that backup and recovery is covered under the BYOI assumption